communications/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+from mongoengine import Document, StringField
+from mongoengine import connect
+
+class ChatRoom(Document):
+    company_id_1 = StringField()
+    company_id_2 = StringField()
+    mongo_room_id = StringField()
```

Comment:
If `company_id_1`, `company_id_2`, and `mongo_room_id` are required logically, define it in the model:
```
company_id_1 = StringField(required=True)
company_id_2 = StringField(required=True)
mongo_room_id = StringField(required=True, unique=True)
```
Adding `unique=True` to `mongo_room_id` may help prevent duplicates if it's meant to be a UUID or slug for the room.